### ISSUE TITLE
No contact showing on main screen

### ISSUE THREAD

**Issue by RDXRiasad** [Bnyro/ConnectYou#152]
### Steps to reproduce

Showing contact bug

### Expected behavior

After entering the app app should show contact right?

### Actual behavior

Not showing anything 
![Screenshot_20230506-091205_Connect You](https://user-images.githubusercontent.com/99120106/236596273-cbe2e7f2-e30d-420d-b214-e5cce0592869.png)

### Connect You version

version 3.1

### Android version

Android 13

### Other details

All permission allowed.

### Acknowledgements

- [x] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
Which app were the contacts created with?

**Comment by RDXRiasad** [comment]
> Which app were the contacts created with?

It's random. some created with Google contact, some created with simple contact and some treated with your app

**Comment by RDXRiasad** [comment]
> > Which app were the contacts created with?
> 
> It's random. some created with Google contact, some created with simple contact and some treated with your app

**Comment by RDXRiasad** [comment]
Any

**Comment by RDXRiasad** [comment]
Any fix?

**Comment by fellinir** [comment]
It seems you got black text. Have you tried tapping on one of those?

**Comment by RDXRiasad** [comment]
Yes i tried. I clicked on them and they are also blank and I also tapped call button nothing happaned. I tried with light mode and dark mode both. I tried importing contact by .Vcf 

Nothing is showing blank as https://user-images.githubusercontent.com/99120106/236596273-cbe2e7f2-e30d-420d-b214-e5cce0592869.png

**Comment by fellinir** [comment]
Do you perhaps have something installed which modifies fonts or touches delicate parts of the system with root privileges?

**Comment by RDXRiasad** [comment]
My font is google sans default font and yes my device is rooted but no system parts touched

**Comment by RDXRiasad** [comment]
![Screenshot_20230527-174351_Connect You](https://github.com/Bnyro/ConnectYou/assets/99120106/e3dfcfeb-2f3c-4590-87ff-8803f7a8d794)

I just found out that after searching contact apearing so it's proved it can access and read contacts. But what's wrong in main page? Not showing anything

**Comment by fellinir** [comment]
So you got access to contacts and probably the number of spheres in the home screen matches the number of contacts anyway.

1. You reported version 3.1. Does this bug occur on 4.0?
1. Maybe try changing font and see if it changes? (it is possible in modern ROMs)

**Comment by RDXRiasad** [comment]
Yes i'm on v4.0 right now. 
I changed font now still same. I tried changing dpi too still same

**Comment by RDXRiasad** [comment]
@Bnyro any fix bruh?

**Comment by RDXRiasad** [comment]
@Bnyro Any fix bruh?

**Comment by fellinir** [comment]
I have no idea what that could be. The last thing I can suggest is to clear app's data and see if something changed.
You can also type here what ROM and phone model you are using to help the developer. Something system side is probably not compatible with the app.

**Comment by RDXRiasad** [comment]
@Bnyro i think the app is not android 14(beta) ready. I'm saw that i cant turn on notification for this app

**Comment by fellinir** [comment]
You said

> @Bnyro i think the app is not android 14(beta) ready. I'm saw that i cant turn on notification for this app

You said it was A13, not a beta A14. Maybe that's the problem.

**Comment by RDXRiasad** [comment]
it's A13 but rom dev updated a lot of things from a14 beta(a14 not released yet)
Just we are seeing some beta things from a14 in a13

**Comment by fellinir** [comment]
Okay so you cannot expect things to work well with such hacked ROM and it is not up to the dev to make an app working for an unreleased Android version. You should close this issue.
